app/views/welcome/show.html.erb

```diff
@@ -2,12 +2,9 @@
   style="background-image: url(<%= asset_pack_path 'media/images/splash.webp' %>); background-position: center; ">
   <div class="container flex-wrap flex-md-nowrap">
     <div class="row">
-      <div class="col-md-10">
+      <div class="col-md-12">
         <h1 class="me-auto">Digital Repository</h1>
       </div>
-      <div class="col-md-2">
-        <a href="#terms-of-deposit" class="terms">Terms of deposit</a>
-      </div>
```

Comment:
we now have this in the header layout across all pages, so we don't need it here on the home page